content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -167,6 +167,39 @@ fail validation.
 				</ul>
 			</td>
 		</tr>
+		<tr>
+			<td style="white-space: nowrap">Probes or Lifecycle Hooks with Host field</td>
+			<td>
+				<p>HostProbeOrHostLifecycle should be disallowed entirely</p>
+				<p><strong>Restricted Fields</strong></p>
+				<ul>
+					<li><code>spec.containers[*].livenessProbe.httpGet.host</code></li>
+					<li><code>spec.containers[*].readinessProbe.httpGet.host</code></li>
+					<li><code>spec.containers[*].startupProbe.httpGet.host</code></li>
+					<li><code>spec.containers[*].livenessProbe.tcpSocket.host</code></li>
+					<li><code>spec.containers[*].readinessProbe.tcpSocket.host</code></li>
+					<li><code>spec.containers[*].startupProbe.tcpSocket.host</code></li>
+					<li><code>spec.containers[*].lifecycle.postStart.tcpSocket.host</code> <small>Deprecated. TCPSocket is <strong>not</strong> supported as a LifecycleHandler and kept for backward compatibility.</small></li>
+					<li><code>spec.containers[*].lifecycle.preStop.tcpSocket.host</code> <small>Deprecated. TCPSocket is <strong>not</strong> supported as a LifecycleHandler and kept for backward compatibility.</small></li>
```

Comment:
the deprecation isn't really relevant for the policy here

```suggestion
					<li><code>spec.containers[*].lifecycle.postStart.tcpSocket.host</code></li>
					<li><code>spec.containers[*].lifecycle.preStop.tcpSocket.host</code></li>
```

Reply:
removed this here and else where